test/nightly/azureResourceMongoDB.test.ts

```diff
@@ -40,27 +44,35 @@ suite('MongoDB action', async function (this: ISuiteCallbackContext): Promise<vo
     });
 
     test('Create Mongo Database', async () => {
-        const collectionName: string = randomUtils.getRandomHexString(12);
-        const testInputs: string[] = [testAccount.getSubscriptionContext().subscriptionDisplayName, `${accountName} (MongoDB)`, databaseName1, collectionName];
+        const collectionName2: string = randomUtils.getRandomHexString(12);
+        const testInputs: string[] = [testAccount.getSubscriptionContext().subscriptionDisplayName, `${accountName} (MongoDB)`, databaseName1, collectionName2];
         await testUserInput.runWithInputs(testInputs, async () => {
             await vscode.commands.executeCommand('cosmosDB.createMongoDatabase');
         });
         assert.ok(await doesMongoDatabaseExist(accountName, databaseName1));
     });
 
     test('Create Mongo Collection', async () => {
-        const databaseName2: string = randomUtils.getRandomHexString(12);
-        const collectionName: string = randomUtils.getRandomHexString(12);
-        const testInputs: string[] = [testAccount.getSubscriptionContext().subscriptionDisplayName, `${accountName} (MongoDB)`, '$(plus) Create new Database...', databaseName2, collectionName];
+        const testInputs: string[] = [testAccount.getSubscriptionContext().subscriptionDisplayName, `${accountName} (MongoDB)`, '$(plus) Create new Database...', databaseName2, collectionName1];
         await testUserInput.runWithInputs(testInputs, async () => {
             await vscode.commands.executeCommand('cosmosDB.createMongoCollection');
         });
         const mongoClient: MongoClient | undefined = await getMongoClient(accountName);
         const listCollections: Collection[] = await mongoClient.db(databaseName2).collections();
-        const collection: Collection | undefined = listCollections.find((collection: Collection) => collection.collectionName === collectionName);
+        const collection: Collection | undefined = listCollections.find((collection: Collection) => collection.collectionName === collectionName1);
```

Comment:
You can replace this line and the lines above with your new `doesMongoCollectionExist` function.